solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -402,6 +447,219 @@
+                "inputs": [
+                    {
+                        "name": "DeviceTelemetry",
+                        "properties": {
+                            "datasource": {
+                                "properties": {
+                                    "consumerGroupName": "[variables('consumerGroupName')]",
+                                    "iotHubNamespace": "[parameters('iotHubName')]",
+                                    "sharedAccessPolicyKey": "[listkeys(variables('iotHubKeyResource'), variables('iotHubApiVersion')).primaryKey]",
+                                    "sharedAccessPolicyName": "[variables('iotHubKeyName')]",
+                                    "endpoint": "messages/events"
+                                },
+                                "type": "Microsoft.Devices/IotHubs"
+                            },
+                            "serialization": {
+                                "properties": {
+                                    "encoding": "UTF8"
+                                },
+                                "type": "Json"
+                            },
+                            "type": "Stream"
+                        }
+                    },
+                    {
+                        "name": "DeviceGroups",
+                        "properties": {
+                            "datasource": {
+                                "properties": {
+                                    "container": "devicegroups",
+                                    "dateFormat": "yyyy-MM-dd",
+                                    "timeFormat": "HH-mm",
+                                    "pathPattern": "referenceinput/{date}/{time}/devicegroups.json",
+                                    "storageAccounts": [
+                                        {
+                                            "accountKey": "[listkeys(variables('storageResourceId'), variables('storageApiVersion')).keys[0].value]",
+                                            "accountName": "[parameters('storageName')]"
+                                        }
+                                    ]
+                                },
+                                "type": "Microsoft.Storage/Blob"
+                            },
+                            "serialization": {
+                                "properties": {
+                                    "fieldDelimiter": ",",
+                                    "encoding": "UTF8"
+                                },
+                                "type": "Csv"
+                            },
+                            "type": "Reference"
+                        }
+                    },
+                    {
+                        "name": "Rules",
+                        "properties": {
+                            "datasource": {
+                                "properties": {
+                                    "container": "rules",
```

Comment:
same comments as above, I assume telemetry agent will need the container name and path pattern

Reply:
same as above.